docs/design.md

```diff
@@ -494,6 +526,13 @@ of 3 is explained below.
         0x8079b62c,
     };
 ```
+---
+***Note***
+Be Aware that the image trailers makes a ending area of image slot unavailable
+for cary the image data. Especially the swap status might be pretty huge,
+For 128 slot's sector with 4-byte alignment it become 1536 B.
```

Comment:
```suggestion
Be aware that the image trailers make the ending area of the image slot unavailable
for carrying the image data. In particular, the swap status size could be huge.
For example, for 128 slot sectors with a 4-byte alignment, it would become 1536 B.
```
Please check if I haven't misunderstood the last sentence, as I'm not sure about "128 slot's sector"